README.md

```diff
@@ -50,6 +50,11 @@ defmodule MyApp.Router do
 end
 ```
 
+### Customizing Traces
+
+Traces can be customized and filtered by passing options to the `use SpandexPhoenix` macro. 
+See the [Hexdocs] documentation on that module for more information.
```

Comment:
Maybe we should link more specifically to https://hexdocs.pm/spandex_phoenix/SpandexPhoenix.html#content here?

Reply:
Right, def should. wasn't sure if there was a special syntax for linking to internal docs w/ hexdocs. i'll fix it up :)